Makefile

```diff
@@ -0,0 +1,31 @@
+.PHONY: deps build run check test clippy clean
+
+TEST_DIR=cairo_programs
+TEST_FILES:=$(wildcard $(TEST_DIR)/*.cairo)
+COMPILED_TESTS:=$(patsubst $(TEST_DIR)/%.cairo, $(TEST_DIR)/%.json, $(TEST_FILES))
+
+$(TEST_DIR)/%.json: $(TEST_DIR)/%.cairo
+	cairo-compile --cairo_path="$(TEST_DIR):$(BENCH_DIR)" $< --output $@
+
+deps:
+	pyenv install pypy3.7-7.3.9
+	pyenv global pypy3.7-7.3.9
```

Comment:
We shouldn't mess with `pyenv global`. Let's use `pyenv local` instead.

Reply:
Changed to local!